src/common/FinanceNavigation/FinanceNavigation.js

```diff
@@ -12,29 +12,29 @@ import { FINANCE_NAVIGATION_TABS } from './constants';
 
 const FinanceNavigation = ({ history, match: { path } }) => {
   const getTabStyle = tabId => (path.includes(`/finance/${tabId}`) ? 'primary' : 'default');
-  const goToTab = (tabId, withSort = true) => history.push(`/finance/${tabId}${withSort ? '?sort=name' : ''}`);
```

Comment:
Could it break default sorting?

Reply:
Nope, default sorting is included in default queey